src/nvim/CMakeLists.txt

```diff
@@ -852,6 +852,42 @@ add_glob_target(
   '
   EXCLUDE ${EXCLUDE_CLANG_TIDY})
 
+add_custom_target(clang-analyzer)
+add_dependencies(clang-analyzer-main copy_compile_commands)
+add_dependencies(clang-analyzer clang-analyzer-main)
+
+function(clang_analyzer)
+  cmake_parse_arguments(ARG
+    ""
+    "CHECK"
+    "EXCLUDE_FILES"
+    ${ARGN})
+
+  add_glob_target(
+    TARGET ${ARG_CHECK}
+    COMMAND ${CLANG_TIDY_PRG}
+    FILES ${NVIM_SOURCES} ${NVIM_HEADERS}
+    FLAGS --quiet
+    --checks='
+    -*,
+    ${ARG_CHECK},
+    '
+    EXCLUDE ${EXCLUDE_CLANG_TIDY} ${ARG_EXCLUDE_FILES})
+
+  add_dependencies(clang-analyzer ${ARG_CHECK})
+  add_dependencies(${ARG_CHECK} copy_compile_commands)
+endfunction()
+
+# TODO(dundargoc): remove exclude files after fixing the corresponding error
+clang_analyzer(CHECK clang-analyzer-core.UndefinedBinaryOperatorResult
+  EXCLUDE_FILES memfile.c marktree.c)
```

Comment:
Where does memfile.c have this warning?

Reply:
Probably an old warning that got removed at some point. Removed, thanks.